styles.css

```diff
@@ -212,6 +212,26 @@ form {
   background-color: var(--color-green);
 }
 
+/* search-sort */
+#search-input {
+  padding-left: 0.7rem;
+}
+
+.search-sort-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  gap: 1.5rem;
+  flex-wrap: wrap;
+}
+
+#sort-select {
+  padding: 0.625rem 0.7rem;
```

Comment:
Usually it's a good idea to keep spacings a multiple of 0.25rem, to make the overall layout more consistent

Reply:
But otherwise, good job for using rem 👍